hq/test/notifications/AnghammaradNotificationsTest.scala

```diff
@@ -0,0 +1,30 @@
+package notifications
+
+import org.scalatest.{FreeSpec, Matchers}
+
+
+class AnghammaradNotificationsTest extends FreeSpec with Matchers {
+  "outdatedCredentialWarning" - {
+    "TODO" ignore {}
```

Comment:
Looks like we didn't write them yet

Reply:
This is the structure, which let us split out cards to fill in the gaps. Some of these are done in open PRs now, some are yet to be written. Those open PRs will be fixed and re-opened against main when this is merged.

Reply:
You're right @jfsoul I didn't write any tests for these methods,  because all they do is string interpolation. Here's the offending PR: https://github.com/guardian/security-hq/pull/358. 

I really should have deleted this test file but must have forgotten! I think this file is not required.